### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by Jak2k** [MM2-0/Kvaesitso#975]
- The top row of icons, which visible without scrolling, doesn't have invisible screen reader labels
- It is not possible to focus a shortcut in that row.

My screen reader is TalkBack.

**Comment by MM2-0** [comment]
Let me know if you find any more missing content descriptions.
